x-pack/plugin/async-search/src/main/java/org/elasticsearch/xpack/search/AsyncSearch.java

```diff
@@ -52,7 +52,7 @@ public List<RestHandler> getRestHandlers(
         Supplier<DiscoveryNodes> nodesInCluster
     ) {
         return Arrays.asList(
-            new RestSubmitAsyncSearchAction(restController.getSearchUsageHolder()),
+            new RestSubmitAsyncSearchAction(restController.getSearchUsageHolder(), searchRestMetrics),
```

Comment:
I'm not sure how to thread the `searchRestMetrics` here...  How are these classes instantiated?  It seems like it's done via config?